x-pack/plugins/apm/common/anomaly_detection.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { TRANSACTION_TYPE } from './elasticsearch_fieldnames';
+
+export interface MaxAnomaly {
+  [TRANSACTION_TYPE]?: string;
+  anomaly_score?: number;
+  actual_value?: number;
+  job_id?: string;
+}
```

Comment:
If this is our interface I think we should stick to our conventions with camelCased fields. And I think it should not be called `MaxAnomaly` because it contains a couple of other things.

```ts
export interface ServiceAnomalyKPIs {
  transactionType?: string;
  anomalyScore?: number;
  actualValue?: number;
  jobId?: string;
}
```